meteor-app/imports/ui/Report_Builder/columnToolBar.tsx

```diff
@@ -57,6 +57,9 @@ export const ColumnToolBar = ({
 			setSymbolState(column.symbol)
 		}
 	}, [column.symbol])
+	useEffect(() => {
```

Comment:
Instead, refactor this useEffect, into the one below 

use [column] at a single dep. and set column.symbol and column.property accordingly

Less use effects = less re-renders of the dom, which is a good thing